srcpkgs/yabar/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'yabar'
+pkgname=yabar
+version=0.4.0
+revision=2
+build_style=gnu-makefile
+make_use_env=yes
+make_build_args="VERSION=$version"
+hostmakedepends="pkg-config glib-devel asciidoc"
+makedepends="alsa-lib-devel gdk-pixbuf-devel libconfig-devel pango-devel xcb-util-wm-devel"
+short_desc="Modern and lightweight status bar for X window managers"
+maintainer='shizonic <realtiaz@gmail.com>'
+license="MIT"
+homepage="https://github.com/geommer/yabar"
+distfiles="https://github.com/geommer/${pkgname}/archive/${version}.tar.gz"
+checksum=f8cc613413328af78944ccb8063a5ba8e6be11797a883c419876e973e102fed0
+
+post_install() {
+	vdoc README.md
+	vconf examples/internal1.config yabar.config
```

Comment:
i see no code that looks into /etc/yabar.config only $HOME/.config/yabar.

shouldn't this be an example config ?